bin/merge_featurecounts.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env/python
+
+import argparse
+import os
+import re
+from collections import defaultdict
+
+
+def do_the_thing(dest_dir,out_file,input_files):
```

Comment:
Ahh, do the _thing_. Now everything is clear! Maybe you could come up with a better function name here ;)